templates/uwsgi.ini.j2

```diff
@@ -22,7 +22,9 @@ static-map={{ superset_wsgi_static_map }}
 {% endif %}
 buffer-size={{ superset_wsgi_buffer_size }}
 reload-on-rss={{ superset_reload_on_rss }}
+{% if superset_version is version_compare('0.35.0', '>=') %}
```

Comment:
Is this only for `0.35.0`?  Curious coz we haven't deployed that version afaik

Reply:
@moshthepitt Yes, the [create_app()](https://github.com/apache/incubator-superset/blame/a9ff51bd6133559c71a43318f46c80e0fe913618/superset/app.py#L50) function  was just introduced recently. Which is what we would normally use `superset.app:create_app()`.